src/bundle.c

```diff
@@ -478,9 +490,33 @@ static void track_installed(const char *bundle_name)
 	free_string(&dst);
 }
 
+static int find_unsafe_to_delete(const void *a, const void *b)
```

Comment:
Avoid having so similar functions with the same name. it took me some time to figure out this function was different than the other.

Reply:
yeah, I used the same name since in both cases the function is for the exact same purpose and they were static functions, so there was no conflicts, so I thought each file could have its implementation... but I can see how that can be confusing, I'll think of a different name for this function.